client/scripts/views/pages/view_proposal/header.ts

```diff
@@ -245,9 +245,9 @@ export const ProposalHeaderThreadLinkedChainEntity: m.Component<{ proposal: Offc
 export const ProposalHeaderThreadLinkedSnapshot: m.Component<{ 
   proposal: OffchainThread, 
 }, { 
-  initialized, 
+  initialized,
   snapshotProposalsLoaded
-  snapshot
+  currentSnapshot
```

Comment:
I'm not sure I understand why this naming convention change was necessary? snapshot either exists or is null. seems a bit clunky.